### PR TITLE
Add bishal7679.json

### PR DESCRIPTION
**Pull request by bishal7679**
## Changes proposed

Added bishal7679.json on public/data directory

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1373"></a>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.68.0",
+      "version": "0.68.2",
```
Comment (emmalearnscode) [inline package-lock.json:9]: Please revert this change 
```suggestion
      "version": "0.68.0",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Your profile looks good but please undo the changes made to the package-lock file.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: package-lock.json (hunk 1 above)
Comment (bishal7679) [reply package-lock.json:9]: okay!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Code looks good to me! 👍  Good to goo! 🚀

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good
